src/test/kotlin/uk/gov/justice/digital/hmpps/whereabouts/services/locationfinder/AppointmentLocationServiceTest2.kt

```diff
@@ -0,0 +1,300 @@
+package uk.gov.justice.digital.hmpps.whereabouts.services.locationfinder
+
+import io.mockk.Matcher
+import io.mockk.MockKMatcherScope
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.whereabouts.dto.prisonapi.ScheduledAppointmentDto
+import uk.gov.justice.digital.hmpps.whereabouts.model.VideoLinkBooking
+import uk.gov.justice.digital.hmpps.whereabouts.repository.VideoLinkBookingRepository
+import uk.gov.justice.digital.hmpps.whereabouts.services.PrisonApiService
+import uk.gov.justice.digital.hmpps.whereabouts.services.vlboptionsfinder.VideoLinkBookingOptions
+import uk.gov.justice.digital.hmpps.whereabouts.services.vlboptionsfinder.VideoLinkBookingOptionsFinder
+import uk.gov.justice.digital.hmpps.whereabouts.services.vlboptionsfinder.VideoLinkBookingSearchSpecification
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.LocalTime
+
+/**
+ * This test class uses Mockk because I couldn't find a way to get mockito or mockitokotlin2 to compare Sequences.
+ * Not even with a custom Argument matcher.
+ * Show me how to do it and I'll revert to Mockito.
+ */
+class AppointmentLocationServiceTest2 {
```

Comment:
I can't help (sorry) but maybe make the class name more descriptive - I guess in this case the name would include "Sequences"?

Reply:
yeah.  Forgot about that.  I think that once the new algorithm is in use this class will become the only AppointmentLocationServicesTest class, but that's a feeble excuse.